index.js

```diff
@@ -7,6 +7,7 @@ const cp = require('child_process')
 const ChildProcess = cp.ChildProcess
 const assert = require('assert')
 const crypto = require('crypto')
+const isWindows = require('is-windows')
```

Comment:
Can we set the IS_WINDOWS constant right here?

Reply:
Maybe use `const isWindows = require('is-windows')()` instead of doing the underscored variable name?

Reply:
Okay, I'll update the PR.

Reply:
Done